.github/ISSUE_TEMPLATE/new-modules.yml

```diff
@@ -0,0 +1,106 @@
+name: Propose a new module
+
+description: Ideas for new content for brainhack school? Submit an issue to discuss with us!
+
+# labels: 
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Guildeline
+
+        We are very excited to see a proposal of a new module!
+
+        The modules are curated resources for the student to gain neuroscience and data science skills.
+
+        Once we evaluate the content, we will ask you to submit a pull request to add the new content.
+
+        We would like to ensure the new module adhere to the following standards:
+
+        - Use free, open source tools: to ensure all students can access the tool.
+        - Python-based tool: the school focuses on Python based content.
+        - Be of high quality: the material or tool should be from a quality source.
+        - Have a broad appeal: a general neuroscience / data science topic suitable for entry level student.
+
+        Please see the existing modules as examples!
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Module info
+
+  - type: textarea
+    attributes:
+      label: Title
+      description: What's the module you would like to add? Please provide a title.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Module Description
```

Comment:
Maybe a section on "content/topics/outline" could be added? However, it could also be part of the description.

Reply:
that's a great addition to guide the submission!